src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capasitu = 16;
+    private float loadFactor = 0.75f;
+    private int size;
+    private Node<K,V>[] table;
+
+    private class Node<K,V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value,Node<K,V> next) {
```

Comment:
The Node class should be static. Non-static inner classes hold an implicit reference to the instance of the outer class. This can lead to memory leaks if the inner class is held in a context different from the outer class. In your case, the Node class doesn't need a reference to the MyHashMap instance, so it's better to make it static.